scripts/hfx_filename

```diff
@@ -51,6 +51,7 @@ def rpc(session_id, request):
         s.close()
 
 def parse_string(txt):
+    assert txt is not None
```

Comment:
Does this fix the original problem? It looks like 'rpc' may return on a non-200 error code, without raising an exception, and it'd instead fail on this assert here then.

Reply:
This is not for the original problem.
The original change is small. Much of the code if for passing the pytype and pylint check added recently.
This is for passing the pytype.